ui/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -27,6 +28,10 @@ Loader {
                      !root.asset.isImage ? roundedIcon : roundedImage
 
     property bool loading: false
+    property bool hoverEnabled: false
+    readonly property bool hovered: (sourceComponent == roundedIcon && item) ?
+                     item.hovered : false
+    signal clicked()
```

Comment:
This is a bit fishy... you declare a signal `clicked()` but then only implement it only for some types/instances of the Loader

Reply:
No, just handle the clicks inside the Loader itself, no need to add it to other types